cmd/servicelog/post.go

```diff
@@ -75,6 +75,12 @@ func newPostCmd() *cobra.Command {
 }
 
 func (o *PostCmdOptions) Init() error {
+	if o.successfulClusters == nil {
+		o.successfulClusters = make(map[string]string)
+	}
+	if o.failedClusters == nil {
+		o.failedClusters = make(map[string]string)
+	}
```

Comment:
Since the function initializes the struct should it just always initialize?

```suggestion
  o.successfulClusters = make(map[string]string)
	o.failedClusters = make(map[string]string)
```